### PR TITLE
Made k3s documentation a little clearer about starting the container

### PR DESCRIPTION
**Pull request by georgecodes**
I also encountered an issue with the Fabric8 client - have not tried with the official client but I imagine the issue remains - regarding the k3s cluster issuing elliptic curve keys.

### CODE REVIEW

File: modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java [hunk 1]
```diff
@@ -34,7 +34,9 @@ public void shouldStartAndHaveListableNode() {
                 .withLogConsumer(new Slf4jLogConsumer(log))
             // }
         ) {
+            // starting_k3s {
```
Comment (kiview) [inline modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java:37]: Maybe it is more aligned with other docs, if we remove the `start()` include and change the title of the code include to `Creating a K3sContainer`.

**Review by kiview — APPROVED** [review]
Thanks for the contribution @georgecodes, I think this is indeed a helpful hint for many users.

File: modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java (hunk 1 above)
Comment (georgecodes) [reply modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java:37]: Possibly, yes. I was really interested in the EC key problem. This was just an afterthought.

File: modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java [hunk 2]
```diff
@@ -32,7 +32,9 @@ public void shouldStartAndHaveListableNode() {
...
```
Comment (eddumelendez) [inline modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java:35]:
```suggestion
```

**Review by eddumelendez — CHANGES_REQUESTED** [review]

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks for your contribution, @georgecodes !

**Pull request merged**